modules/reactor/src/meshgenerators/SimpleHexagonGenerator.C

```diff
@@ -125,5 +151,6 @@ SimpleHexagonGenerator::generate()
         _external_boundary_name;
   }
 
+  mesh->prepare_for_use();
```

Comment:
why do you need this?

Reply:
We expect the mesh produced by this MG will be used by `PatternedHexMeshGenerator` as input. `PatternedHexMeshGenerator` uses `libMesh::ReplicatedMesh::stitch_meshes()` for mesh stitching. In debug mode, `libMesh::ReplicatedMesh::stitch_meshes()` checks whether `_is_prepared` data of the input mesh. I use `prepare_for_use()` here just to set `_is_prepared` as true to avert the error in debug mode. From exodus file, it seems not make a difference with or without this line. Another option is to put `prepare_for_use()` in `PatternedHexMeshGenerator` right before calling `libMesh::ReplicatedMesh::stitch_meshes()`.

Reply:
ah ok. Maybe it needs a renumbering or the cell neighbors for stitching. i dont really see why right now